templates/order/details.html

```diff
@@ -61,6 +62,39 @@ <h2 class="order-details__header">
         {% endif %}
       {% endif %}
       {% include "order/_ordered_items_table.html" %}
+      {% if order.status == 'open' %}
+        {% if notes %}
+          <h2 class="order-notes__header">{% trans "Notes" context "Order notes list" %}</h2>
+          {% for note in notes %}
+            {% if note.is_public %}
```

Comment:
Can't we just filter only the public notes in the view? For that purpose, we could even have a custom queryset and access the notes like: `order.notes.public()`

Reply:
Yes, please. Currently we're counting on the template for security.

Reply:
It is already being filtered in the view, I just forgot to update template.